### DISCUSSION TITLE
Error Knitting to PDF in R Studio

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#696] [Q&A] [answered]
Saya mengalami error ketika melakukan knit ke pdf
<img width="659" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/cc2006d7-f4cf-4662-b2ee-eee50edb5e15">

Mohon bantuannya kak, terima kasih 🙏

**Comment by kinantytasyao** [comment]
Apakah ketika knit Bapak sudah memilih yang "knit to pdf" bukan "knit" saja Pak?

**Reply by yohanespm77** [reply]
Sudah mba, knit to pdf, tapi muncul error seperti di atas.

**Reply by kinantytasyao** [reply, marked as answer]
Bisa coba diinstall terlebih dahulu package2 yang disebutkan di errornya Pak Yohanes, lalu coba knit to pdf lagi

**Reply by IrfanChairurrachman** [reply]
Dear @yohanespm77 ,

Untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah lebih dari 7 hari respon terakhir. Semoga dapat dipahami. Terima kasih
